clients/src/main/java/org/apache/kafka/clients/consumer/internals/DefaultBackgroundThread.java

```diff
@@ -71,18 +72,21 @@ public class DefaultBackgroundThread extends KafkaThread {
     private final RequestManagers requestManagers;
 
     // Visible for testing
+    @SuppressWarnings("checkstyle:parameternumber")
```

Comment:
Interested to see what the AK committers prefer, but I do prefer this over the suppression.xml for individual functions because it is easier to maintain if ever we remove params, for instance. I find that whatever we add to the suppressions xml will stay there forever